tests/e2e/vsan_stretched_cluster.go

```diff
@@ -163,6 +169,12 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}
 
+		if rwxAccessMode {
+			accessMode = rwx
+		} else {
+			accessMode = ""
```

Comment:
what this if/else is doing exactly ?

Reply:
If accessmode is ReadWriteMany then it uses otherwise ""  in every util , if access mode is blank we will be considering ReadWriteOnce